jupyterhub/app.py

```diff
@@ -606,6 +606,31 @@ def _authenticator_default(self):
         """
     ).tag(config=True)
 
+    throttle_retry_suggest_min = Integer(
```

Comment:
nit: can we make this a tuple instead of two integers? Saves some repeated docs and wouldn't have made me initially think that this is in min(utes) instead of the min(imum).

Reply:
If we anticipate these values changing on the fly while running, the tuple's immutability might be a downside.

Perhaps:

`spawn_throttle_retry_wait_floor` and `spawn_throttle_retry_wait_ceiling`